packages/angular/build/src/tools/babel/plugins/pure-toplevel-functions.ts

```diff
@@ -32,6 +32,18 @@ function isTslibHelperName(name: string): boolean {
   return tslibHelpers.has(originalName);
 }
 
+const babelHelpers: string[] = ['_defineProperty'];
```

Comment:
Prefer using a `Set` over an array for constant access when you need unique values and efficient lookups.
```suggestion
const babelHelpers = new Set<string>(['_defineProperty']);
```